src/test/java/de/doubleslash/keeptime/common/DateFormatterTest.java

```diff
@@ -28,6 +28,7 @@
 public class DateFormatterTest {
 
    @Test
+   @Disabled
```

Comment:
why do we need to disable it now?

Reply:
I thought it got executed although the class was Disabled.  But I tested it and now it works again